fe/fe-core/src/main/java/org/apache/doris/resource/resourcegroup/ResourceGroup.java

```diff
@@ -141,6 +201,23 @@ private static void checkProperties(Map<String, String> properties) throws DdlEx
                 throw new DdlException("The value of '" + ENABLE_MEMORY_OVERCOMMIT + "' must be true or false.");
             }
         }
+
+        // check queue property
+        if (properties.containsKey(MAX_CONCURRENCY)) {
+            if (Integer.parseInt(properties.get(MAX_CONCURRENCY)) < 0) {
```

Comment:
Integer.parseInt may throw an exception, make sure the upper layer catches this exception and gives a user-friendly message